highfidelity/hifi
# MS9539: Update Controls Reference pages

- Fix MS bug [#9539](https://highfidelity.manuscript.com/f/cases/9539/MSFT-HMD-Help-should-display-the-correct-images-and-help-for-the-MS-HMD-Hand-contollers)

<strike>**Test Plan**
In Windows MR:
- Launch Interface
- Open `Help > Controls Reference`
- Controls for Windows MR should show up

In other display modes:
- Launch Interface
- Open `Help > Controls Reference`
- Controls for other display modes should show up</strike>
